Packs/CortexXDR/ReleaseNotes/6_1_78.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Reverting the de-duplication mechanism added in version `6.1.73`.
```

Comment:
This RN should be "fixed an issue where - {the issue}"
Additionally, add a note inside 6.1.73 RN that the version is invalid (like the example I sent on slack)